mvlearn/decomposition/groupica.py

```diff
@@ -224,6 +224,13 @@ def transform(self, Xs, y=None):
         y : None
             Ignored variable.
 
+        index: None, or int or array-like
+            int or list of ints specifying the indices of the
+            inputted views relative to the fitted views.
+            If None, there should be as many inputted views as fitted views.
+            Note that the index parameter is not available in all methods of
+            mvlearn yet.
```

Comment:
```suggestion
        index: int or array-like, default=None
            The index or list of indices of the fitted views to which the
            inputted views correspond. If None, there should be as many
            inputted views as the fitted views and in the same order.
            Note that the index parameter is not available in all methods of
            mvlearn yet.
```

Just want to get the language right here, as it will be what I use for everything. If looks good to you, add to everywhere else in this PR please.